packages/harmony/src/components/expandable-nav-item/types.ts

```diff
@@ -0,0 +1,19 @@
+import type { ReactNode } from 'react'
+
+import type { WithCSS } from '../../foundations/theme'
+import type { IconComponent } from '../icon'
+
+export type ExpandableNavItemProps = WithCSS<{
```

Comment:
heads up for you, the WithCSS prop should not typically be used, emotion is smart enough to take incoming css prop and convert it to "className"

what this means for ExpandableNavItemProps you can do
```
type ExpanableNaviItemProps = {className?: string ,....}
```
and that will make it so callers of this component can do
```
<ExpandableNaviItem css={{...}} />
```

WithCSS is there as a stop-gap for certain types of prop usages, like passing an object of props into a component object like "<ComplexComponent PaperProps={{prop1, prop2, css: {...}} />"